src/test/java/org/folio/linked/data/integration/kafka/listener/SourceRecordDomainEventListenerIT.java

```diff
@@ -42,6 +45,7 @@ static void setup(@Autowired KafkaAdminService kafkaAdminService) {
     kafkaAdminService.createTopics(TENANT_ID);
   }
 
+  @Disabled
```

Comment:
There are sonar violations for these disabled tests.

Reply:
fixed